pandas/io/clipboard.py

```diff
@@ -3,12 +3,13 @@
 from pandas.compat import StringIO
 
 
-def read_clipboard(**kwargs):  # pragma: no cover
+def read_clipboard(sep='\s+', delim_whitespace=False, **kwargs):  # pragma: no cover
     """
     Read text from clipboard and pass to read_table. See read_table for the
     full argument list
 
-    If unspecified, `sep` defaults to '\s+'
+    If unspecified, `sep` defaults to '\s+' and delim_whitespace defaults
```

Comment:
Take a look at the [NumPy docstring reference](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt), which we try to follow. So this would be something like like


```
sep : str, optional
    A string or regex delimiter. Default `'\s+'` 
delim_whitespace: bool, optional
    Whether to .... Default False
```